sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -344,8 +339,8 @@
      *
      * <pre>
```

Comment:
should we no more have `<pre>` tags but instead use the codesnippets file ?

Reply:
There is an issue filed as part of KV cleanup. Will do this in a separate PR.